python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -189,9 +218,18 @@ def stop(self):
 
     @property
     def stopped(self):
+        """
+        Returns if the flusher has stopped as a boolean?
+        """
         return self._stop.isSet()
 
     def join(self, *args, **kwargs):
+        """
+        Not clear if this is called anywhere
```

Comment:
I'm not sure it needs much of a comment. It's a subclass of `threading.Thread` and this is the [`join` method](https://docs.python.org/2.7/library/threading.html?highlight=threading.thread.join#threading.Thread.join). The only potential addition is that is requests the loop to stop running.